src/app/qan-edit-column/qan-edit-column.component.ts

```diff
@@ -0,0 +1,93 @@
+import {Component, OnDestroy} from '@angular/core';
+import {QueryTableConfigurationService} from '../query-profile/query-table-configuration.service';
+
+@Component({
+  selector: 'app-qan-edit-column',
+  templateUrl: './qan-edit-column.component.html',
+  styleUrls: ['./qan-edit-column.component.scss']
+})
+export class QanEditColumnComponent implements OnDestroy {
+
+  public isConfigurationMenu = false;
+  public columns: any;
+  private subscription: any;
+  public isMainChecked: boolean;
+  public isSubMain: boolean;
+  public isAllUnchecked: boolean;
+  private ids = {
+    load: 'load',
+    count: 'count',
+    latency: 'latency'
+  };
+  private configurations = {
+    loadOptions: ['sparkline', 'value', 'percentage'],
+    countOptions: ['sparkline', 'value', 'percentage', 'queriesPerSecond'],
+    latencyOptions: ['sparkline', 'value', 'distribution']
+  };
+
+  constructor(public configService: QueryTableConfigurationService) {
+    this.subscription = this.configService.source.subscribe(items => {
+      this.columns = items;
+    });
+  }
+
+  undisableOptions(id, key) {
+    this.configService.toggleConfig(id, key);
+    switch (id) {
+      case this.ids.load:
+        this.isAllUnchecked = this.columns[0].sparkline || this.columns[0].value || this.columns[0].percentage;
```

Comment:
what do you think to rename `isAllUnchecked` to `isAnyChacked` - IMHO it is a little bit better describes purpose